packages/job-worker/src/playout/abPlayback/index.ts

```diff
@@ -103,3 +122,36 @@ export function applyAbPlaybackForTimeline(
 
 	return newAbSessionsResult
 }
+
+function findPlayersInRouteSets(routeSets: Record<string, StudioRouteSet>): MembersOfRouteSets[] {
```

Comment:
perhaps this could produce a structure which will make for quicker lookups? and avoid the loop within loop inside `abPoolFilterDisabled`
could be `Map<poolName, Set<playerId>`, then it would be a pair of hash backed lookups instead of an iteration.

But also, this array shouldnt be large enough for this to be a problem, so not a big problem

Reply:
Yes, we talked about it back when it was made, but agreed on that this was such a small iteration that it would be fine to do this way.